example/lib/ios_popover/_ios_popover_menu.dart

```diff
@@ -0,0 +1,222 @@
+import 'dart:math';
+
+import 'package:flutter/material.dart';
+
+class IosPopoverMenu extends StatelessWidget {
+  const IosPopoverMenu({
+    super.key,
+    this.radius = const Radius.circular(12),
+    this.arrowWidth = 18.0,
+    this.arrowLength = 12.0,
+    required this.arrowDirection,
+    this.backgroundColor = const Color(0xFF474747),
+    this.padding = EdgeInsets.zero,
+    this.arrowFocalPoint,
+    required this.child,
+  });
+
+  /// Radius of the decoration corners.
+  final Radius radius;
+
+  /// Extent of the arrow.
+  final double arrowLength;
+
+  /// Width of the arrow.
+  final double arrowWidth;
+
+  /// Direction where the arrow points to.
+  final ArrowDirection arrowDirection;
+
+  /// Padding around the popover content.
+  final EdgeInsets padding;
+
+  /// Color of the decoration.
```

Comment:
Notice that there's nothing called a "decoration" in the public API for this widget.

Reply:
Updated.